patroni/postgresql/__init__.py

```diff
@@ -39,14 +40,24 @@
 
 logger = logging.getLogger(__name__)
 
-STATE_RUNNING = 'running'
-STATE_REJECT = 'rejecting connections'
-STATE_NO_RESPONSE = 'not responding'
-STATE_UNKNOWN = 'unknown'
-
 STOP_POLLING_INTERVAL = 1
 
 
+class PgIsReadyStatus(str, Enum):
```

Comment:
IMO, it should be `IntEnum`. String values aren't used anywhere in the code for reporting, only for comparing it with other strings.
As a result, we don't need to have mapping [here](https://github.com/patroni/patroni/pull/3293/files#diff-72030eab48ce77175c998a2fd1ee9e84fe950e132b66972aa72496532a3923e5R334-R338).
And, actually, `pg_isready(self)` should be returning `PgIsReadyStatus`, not `str`.